lib/application.js

```diff
@@ -89,13 +95,32 @@ app.callback = function(){
   var self = this;
 
   return function(req, res, next){
-    var ctx = new Context(self, req, res);
+    var ctx = self.createContext(req, res);
 
     co.call(ctx, gen)(next || ctx.onerror);
   }
 };
 
 /**
+ * Initialize a new context.
+ *
+ * @api private
+ */
+
+app.createContext = function(req, res){
+  var context = Object.create(this.context);
```

Comment:
Why is the `Object.create` called again? The object with that prototype was created on L44.

I tested it and it seems to work without the second creating well.

The same applies to request and response.

(I was looking at the code, found this line's history and went to this PR.)